modules/ROOT/pages/bonita-purpose.adoc

```diff
@@ -1,86 +1,37 @@
+:description: This page gives an overview of the values of Bonita Platform, as well as the business use-cases Bonita helps solve, both through types of automation projects and industry verticals.
 
-Bonita is an open-source and extensible platform for business process automation and optimization. Bonita Platform accelerates the development and production of business applications, with clear separation between capabilities for visual programming and for coding. +
-BPMN (Business Process Modeling Notation), full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. +
-Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
+{description}
 
-Bonita Platform has two components: the development environment, Bonita Studio, and the runtime environment, Bonita Runtime.
+== Values of Bonita Platform
 
-== What can I do with Bonita?
+As a platform, Bonita supports your automation project from implementation to maintenance and evolution, with a set of dedicated tools. +
+It comes with plenty of provided artifacts, speeding up the development, facilitating the communication among team members, but also easing the maintenance. +
+As an open platform, it encourages the creation of more artifacts to perfectly fit the business and technical needs of a given automation project. As our customers say; "you are never stuck with Bonita". +
+Moreover, the built-in BPM engine provides events traceability upon which monitoring and reporting dashboards are created, providing in-depth visibility and undersanding of the process execution. 
+Plus, its robustness enables the BPM engine to handle a company's critical projects, with high performance and technical monitoring indicators.
 
-Bonita is a Digital Process Automation (DPA) platform that allows you to create applications based on processes. Processes can be partially automated (if involving users) or fully automated.
+== Types of automation projects
 
-DPA, available with all Bonita editions, is applicable to a wide range of use cases: applications can go from loan request management, to student application processes, to analytical processes in drug discovery - and more! You can find the various industries where Bonita is used as well as use cases from our https://www.bonitasoft.com/industries[website dedicated page].
+=== The need for automation
+Some business processes do not need to be automated. Their execution stays in the scope of one team, the data they handle does not need to be retrieved nor sent from/to many places, nor undergo transformations from one step to the other, the execution of each instance does not need traceability nor reminders, and there is no need of reporting about their execution. +
```

Comment:
This sentence is way to long, in a fully negation style, hard to follow IMHO. I suggest to split it.

Reply:
ok. Working on it.